src/main/java/com/okta/sdk/framework/ApiClient.java

```diff
@@ -64,9 +61,20 @@
     ////////////////////////////////////////////
 
     public ApiClient(ApiClientConfiguration config) {
+        Proxy proxy = ProxySelector.getDefault().select(URI.create(config.getBaseUrl())).iterator().next();
+        URI proxyUri = URI.create(proxy.type() + "://" + proxy.address());
+        HttpRoutePlanner routePlanner;
+        if (Proxy.Type.HTTP.equals(proxy.type())) {
```

Comment:
The proxy selector should already select the proxy based on the URI, why here check the protocol again?